hack/drone.jsonnet

```diff
@@ -179,11 +179,9 @@ local apid = Step("apid", depends_on=[setup_ci]);
 local osctl_linux = Step("osctl-linux", depends_on=[setup_ci]);
 local osctl_darwin = Step("osctl-darwin", depends_on=[setup_ci]);
 local docs = Step("docs", depends_on=[osctl_linux]);
-local integration_test = Step("integration-test", depends_on=[setup_ci]);
```

Comment:
These changes are necessary to prevent the app tarballs from being corrupted. Each on of these steps now depends on `apps`, and the `apps` target outputs tarballs. If all run in parallel then they start to compete in the write and subsequent steps end up getting corrupted tarballs